aten/src/ATen/quantized/Quantizer.h

```diff
@@ -225,4 +225,6 @@ CAFFE2_API Tensor new_qtensor(
     const TensorOptions& options,
     QuantizerPtr quantizer);
 
+CAFFE2_API void set_quantizer_(const Tensor& self, ConstQuantizerPtr quantizer);
```

Comment:
FWIW, you didn't have to funcify this; you could have just readded the method to manual TensorBody.h. But this is fine too.